src/render/blog/hello-rest-service.html.md

```diff
@@ -14,53 +14,408 @@ This article concentrates on integration with service using Knot.x, instead of c
+</blockquote>
+
+As we already have base Knot.x setup, let’s start adopting it to our needs. 
+If you open `knotx-standalone-1.0.0.json` file in text editor, you can find that it contains array 
+with list of modules. These are the [Knot.x modules](https://github.com/Cognifide/knotx/wiki/KnotxModules) 
+that are going to be started.
+
+```json
+{
+  "modules":[
+    "knotx:io.knotx.KnotxServer",
+    "knotx:io.knotx.HttpRepositoryConnector",
+    "knotx:io.knotx.FilesystemRepositoryConnector",
+    "knotx:io.knotx.FragmentSplitter",
+    "knotx:io.knotx.FragmentAssembler",
+    "knotx:io.knotx.ServiceKnot",
+    "knotx:io.knotx.ActionKnot",
+    "knotx:io.knotx.HandlebarsKnot",
+    "knotx:io.knotx.HttpServiceAdapter"
+  ]
 }
 ```
-## Run "dynamic" page
-So, now it’s time to run our new "dynamic" page. Just start Knot.x using `./run.sh` command and go to `http://localhost:8092/html/books.html` in your favourite browser.
 
+All of the above modules are explained on the project [Wiki](https://github.com/Cognifide/knotx/wiki). 
+In this tutorial, we’re going to pay attention only about a few of them.
+
+In order to change default configuration of any of the module, you need to modify 
+this configuration JSON by adding `config` JsonObject for the module(s) you want to modify, e.g.:
+
+```json
+ "modules":[..],
+ "config": {
+   "knotx:io.knotx.KnotxServer": {
+     "options": {
+        "config": {
+           ...
+        }
+      }
+    }
+  }
+```
+
+You can find more details on how to configure Knot.x on [Wiki – How to Configure](https://github.com/Cognifide/knotx/wiki/KnotxDeployment#how-to-configure-).
+
+### Repository configuration
+
+As mentioned before, we are going to fetch HTML template from our local file system. 
+Knot.x is shipped with a dedicated module called [FileSystemRepositoryConnector](https://github.com/Cognifide/knotx/wiki/FilesystemRepositoryConnector)
+for this purpose. You need set up the connector to use `/library` folder from where requested HTML files 
+(consisting of handlebars snippets) will be fetched.
+
+```json
+  "modules": [...],
+  "config": {
+    "knotx:io.knotx.FilesystemRepositoryConnector": {
+      "options": {
+        "config": {
+          "catalogue": "./library/"
+        }
+      }
+    }
+  }
+```
+
+Different repository connectors can be used, so we need to somehow tell Knot.x which one should be used. 
+You can define a condition when our repository should be used, and the condition is simply a RegExp of the requested URI.
+
+Our template file (books.html) is available in `html` folder, under `library` root folder. 
+It means, that our target is that whenever user does `GET /html/books.html` request, 
+the Knot.x should look for the `/html/books.html` file in `/library` folder on local file system.
+In order to achieve that, we need to tell KnotxServer module that:
+
+<blockquote>
+Any request to <strong>/html/.*</strong> should fetch template from file system repository.
```

Comment:
and instead of strong make like this: (it was strong as on WP we didn't have such nice stuff :) )
```
`/html/.*`
```